doc/papers/joss/paper.bib

```diff
@@ -0,0 +1,75 @@
+@article{Brown:2010,
+    Adsurl = {https://doi.org/10.1007/s10915-010-9396-8},
+    Author = {{Brown}, J.},
+    Journal = {Journal of Scientific Computing},
+    Volume = 45,
+    Month = oct,
+    Title = {{Efficient Nonlinear Solvers for Nodal High-Order Finite Elements in 3D}},
+    Year = 2010,
+    doi = {10.1007/s10915-010-9396-8},
+}
+
+@misc{CUDAwebsite,
+    title = "https://developer.nvidia.com/about-cuda",
+    url = "https://developer.nvidia.com/about-cuda",
+    year = "2020"
+}
+
+@misc{libceed-dev-site,
+  title =  {lib{CEED} development site},
+  url =    {https://github.com/ceed/libceed},
+  year = 2020
+}
+
+@misc{LIBXSMM,
+    title = "{LIBXSMM development site}",
+    url = "http://github.com/hfp/libxsmm",
+    year = "2020"
+}
+
+@misc{MAGMAwebsite,
+    title = "{MAGMA development site}",
+    url = "https://bitbucket.org/icl/magma",
+    year = "2020"
+}
+
+@misc{MFEMlibrary,
+    title = "{MFEM}: {Modular Finite Element Methods Library}",
+    url = "https://mfem.org/",
+    year = "2020",
+    doi = "10.11578/dc.20171025.1248"
+}
+
+@misc{Nekwebsite,
+    title = "Nek5000",
+    url = "https://nek5000.mcs.anl.gov/",
+    year = "2020"
+}
+
+@misc{OCCAwebsite,
+    title = "{OCCA} development site",
+    url = "http://github.com/libocca/occa",
+    year = "2020"
+}
+
+@article{Orszag:1980,
+    title = "Spectral methods for problems in complex geometries",
+    journal = "Journal of Computational Physics",
+    volume = "37",
+    pages = "70 - 92",
+    year = "1980",
+    doi = "https://doi.org/10.1016/0021-9991(80)90005-4",
+    author = "Steven A Orszag"
+}
+
+@TechReport{PETScUserManual,
+  author = {Satish Balay and Shrirang Abhyankar and Mark~F. Adams and Jed Brown and Peter Brune
+            and Kris Buschelman and Lisandro Dalcin and Alp Dener and Victor Eijkhout and William~D. Gropp
+            and Dmitry Karpeyev and Dinesh Kaushik and Matthew~G. Knepley and Dave~A. May and Lois Curfman McInnes
+            and Richard Tran Mills and Todd Munson and Karl Rupp and Patrick Sanan
+            and Barry~F. Smith and Stefano Zampini and Hong Zhang and Hong Zhang},
+  title  = {{PETS}c Users Manual},
+  institution = {Argonne National Laboratory},
+  year   = 2020,
+  number = {ANL-95/11 - Revision 3.13}
+}
```

Comment:
Cite the BP paper, https://journals.sagepub.com/doi/full/10.1177/1094342020915762, and the CEED project website, https://ceed.exascaleproject.org ?